agent/proto/agent.proto

```diff
@@ -53,6 +53,7 @@ message WorkspaceAgentScript {
 	bool run_on_stop = 6;
 	bool start_blocks_login = 7;
 	google.protobuf.Duration timeout = 8;
+	string display_name = 9;
```

Comment:
If we're changing the agent API, it's possible that this value will be blank when the agent and the controlplane are out of sync.

@mafredri could you speak more to how we handle agent versioning? This area is new to me, too.

Reply:
Yes, appending fields works wrt backwards compatibility but a mis-match of agent/server means that either the fields won't be populated or it won't be parsed. For this feature we should handle the case where the agent doesn't set these fields.

(And maybe we do, starting to review the code now.)

Reply:
After reviewing I don't think we need to worry about this. If the agent is old it won't send any timing data. Likewise if it's new the endpoint won't exist. In either case there isn't anything we can do.